backend/spec/requests/answers_request_spec.rb

```diff
@@ -80,5 +80,35 @@
         end
       end
     end
+
+    context 'when data is not valid', bullet: :skip do
```

Comment:
why did you skip bullet? is this test raising something for bullet?
If yes, this means that in production it would also raise, are you considering ignoring it?